kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/CreateTopicRequest.java

```diff
@@ -48,14 +53,17 @@ public static Builder builder() {
   @JsonCreator
   static CreateTopicRequest fromJson(
       @JsonProperty("topic_name") String topicName,
-      @JsonProperty("partitions_count") @Nullable  Integer partitionsCount,
+      @JsonProperty("partitions_count") @Nullable Integer partitionsCount,
       @JsonProperty("replication_factor") @Nullable Short replicationFactor,
+      @SuppressWarnings("checkstyle:LineLength")
```

Comment:
Don't suppress the warning. Break the line. I think breaking it anywhere is valid, either between the annotations, after the annotations or after the type.